html/semantics/forms/form-submission-0/submission-checks.window.js

```diff
@@ -0,0 +1,35 @@
+async_test(t => {
+  const frame = document.createElement("frame"),
+        form = document.createElement("form");
+  t.add_cleanup(() => frame.remove());
+  form.action = "/common/blank.html";
+  form.target = "doesnotmattertwobits";
+  frame.name = "doesnotmattertwobits";
+  document.body.appendChild(frame);
+  frame.onload = t.step_func(() => {
+    if(frame.contentWindow.location.href === "about:blank")
+      return;
+    assert_unreached();
```

Comment:
So the origial navigation to /common/blank.html is expected to be interrupted but the disconnected form submission, but that submission should not actually cause any other navigation to occur either?